opencti-platform/opencti-front/src/private/components/analyses/opinions/StixCoreObjectOpinionsRadar.tsx

```diff
@@ -63,12 +64,44 @@ const StixCoreObjectOpinionsRadar: FunctionComponent<StixCoreObjectOpinionsRadar
   ];
   const labels = opinionOptions.map((m) => m.label);
   const colors = generateGreenToRedColors(opinionOptions.length);
+
+  const handleRadarOpen = () => {
+    if (opinionsDistribution && opinionsDistribution.length > 0) {
+      handleOpen();
+    } else {
+      MESSAGING$.notifyError(
+        <span>
+          {t_i18n('No opinions have been added for this entity.')}
+        </span>,
+      );
+    }
+  };
+
+  if (opinionOptions.length === 0) {
+    return (
+      <Chart
+        options={{
+          noData: {
+            text: t_i18n('No data available.'),
```

Comment:
When I click on "No data available" I have this error appearing:
![image](https://github.com/user-attachments/assets/100391aa-0416-47ed-a87e-610edbfda209)

Which is same behavior as when we have a graph, I think we could get rid of this on click event in this case. If easy to do, otherwise forget it